src/gis.cc

```diff
@@ -0,0 +1,190 @@
+#include "gis.h"
+
+using namespace std;
+
+namespace cyclus {
+
+GIS::GIS() {
+  latitude = 0.0;
+  longitude = 0.0;
+}
+
+GIS::GIS(float decimal_lat, float decimal_lon) {
```

Comment:
Another option is to provide default values for this overloaded constructor, so that you don't need a default constructor (GIS()) at all. 

http://stackoverflow.com/questions/187640/default-parameters-with-c-constructors

```
GIS::GIS(float decimal_lat=0, float decimal_lon=0) {
  latitude = setPrecision(decimal_lat * 3600, 1);
  longitude = setPrecision(decimal_lon * 3600, 1);
}
```